addons/cargo/CfgEventHandlers.hpp

```diff
@@ -88,4 +88,14 @@ class Extended_InitPost_EventHandlers {
             init = QUOTE(_this call DFUNC(initObject));
         };
     };
+    class RoadCone_F {
+        class ADDON {
+            init = QUOTE(_this call DFUNC(initObject));
+        };
+    };
+    class RoadCone_L_F {
```

Comment:
This class is insie `ReammoBox_F` class which already has XEH in this file, no need to have it twice.